plugins/modules/k8s_info.py

```diff
@@ -198,6 +207,7 @@ def argspec():
             namespace=dict(),
             label_selectors=dict(type="list", elements="str", default=[]),
             field_selectors=dict(type="list", elements="str", default=[]),
+            hidden_fields=dict(type="list", elements="str"),
```

Comment:
```suggestion
            hidden_fields=dict(type="list", elements="str", default=[]),
```

Reply:
Again, I don't feel this one is necessary

Reply:
Ah, I found the failing test - I've removed the documented default instead.